articles/app-service-web/nodejs-send-email-with-sparkpost.md

```diff
@@ -0,0 +1,412 @@
+
+```azurecli
+az appservice web create --name <app_name> --resource-group testResource --plan sparkpost_nodejs
+```
+
+Replace `<app_name>` with your own app name. As this value will be used as the default DNS site for the web app, it needs to be unique across all apps in Azure.
+
+A successful request will respond with something like following.
+
+```json
+{
+  "availabilityState": "Normal",
+  "clientAffinityEnabled": true,
+  "clientCertEnabled": false,
+  "cloningInfo": null,
+  "containerSize": 0,
+  "dailyMemoryTimeQuota": 0,
+  "defaultHostName": "<app_name>.azurewebsites.net",
+  "enabled": true,
+  "enabledHostNames": [
+    "<app_name>.azurewebsites.net",
+    "<app_name>.scm.azurewebsites.net"
+  ],
+  "gatewaySiteName": null,
+  "hostNameSslStates": [
+    {
+      "hostType": "Standard",
+      "name": "<app_name>.azurewebsites.net",
+      "sslState": "Disabled",
+      "thumbprint": null,
+      "toUpdate": null,
+      "virtualIp": null
+    },
+    {
+      "hostType": "Repository",
+      "name": "<app_name>.scm.azurewebsites.net",
+      "sslState": "Disabled",
+      "thumbprint": null,
+      "toUpdate": null,
+      "virtualIp": null
+    }
+  ],
+  "hostNames": [
+    "<app_name>.azurewebsites.net"
+  ],
+  "hostNamesDisabled": false,
+  "hostingEnvironmentProfile": null,
+  "id": "/subscriptions/6c6fe2eb-ac4c-4944-a320-203a08a12c1a/resourceGroups/testResource/providers/Microsoft.Web/sites/<app_name>",
+  "isDefaultContainer": null,
+  "kind": "app",
+  "lastModifiedTimeUtc": "2017-05-04T19:43:55.150000",
+  "location": "West US",
+  "maxNumberOfWorkers": null,
+  "microService": "false",
+  "name": "<app_name>",
+  "outboundIpAddresses": "retracted",
+  "premiumAppDeployed": null,
+  "repositorySiteName": "<app_name>",
+  "reserved": true,
+  "resourceGroup": "testResource",
+  "scmSiteAlsoStopped": false,
+  "serverFarmId": "/subscriptions/6c6fe2eb-ac4c-4944-a320-203a08a12c1a/resourceGroups/testResource/providers/Microsoft.Web/serverfarms/sparkpost_nodejs",
+  "siteConfig": null,
+  "slotSwapStatus": null,
+  "state": "Running",
+  "suspendedTill": null,
+  "tags": null,
+  "targetSwapSlot": null,
+  "trafficManagerHostNames": null,
+  "type": "Microsoft.Web/sites",
+  "usageState": "Normal"
+}
+```
+
+You should be able to browse your newly created web application using the following URL.
+
+```
+http://<app_name>.azurewebsites.net
+```
+
+![default-app][default-app]
+
+This is default web app in Azure. We'll now configure it to use Node.js and then push our sample app.
+
+## Set SparkPost API Key in environment variable
+In an earlier step, we've created API Key. Now we'll save this API Key to application's settings which will be available to our codes via environment variables. In our code we've already referenced to this environment variable, hence, it should work without any further modification in code.
```

Comment:
"we've created **an** API Key"
"already referenced this environment variable" (remove **to**)